learn/get-started/deploy-ballerina-on-kubernetes.md

```diff
@@ -14,41 +14,49 @@ redirect_from:
     - /learn/getting-started/deploying-ballerina-on-kubernetes
 ---
 
-In this section, you will be writing a simple Ballerina service and then, you will dockerize the application and deploy it in Kubernetes.
+Code to Cloud is a compiler extension, which is packed with Ballerina. It makes it easier to generate artifacts required for the cloud from your Ballerina code. Currently, you could generate Docker and Kubernetes artifacts from the Ballerina code. This process encourages you to write cloud-ready code from day one without any additional effort. 
```

Comment:
```suggestion
Code to cloud is a compiler extension which is packed with Ballerina. It makes it easier to generate artifacts required for the cloud from your Ballerina code. Currently, you could generate Docker and Kubernetes artifacts from the Ballerina code. This process encourages you to write cloud-ready code from day one without any additional effort. 
```

Reply:
Isn't the word `it` repetitive in the `It makes it easier to generate artifacts required...` sentence?

Reply:
The comma before the word `which` is r grammatically required when a noun is described further. 

Will fix the repetition of the word `it` as `It eases generating the artifacts required for the cloud from your Ballerina code.`

Reply:
Updated.